federation/pkg/federation-controller/namespace/namespace_controller_test.go

```diff
@@ -153,10 +132,12 @@ func TestNamespaceController(t *testing.T) {
 	ns1.DeletionTimestamp = &metav1.Time{Time: time.Now()}
 	namespaceWatch.Modify(&ns1)
 	assert.Equal(t, ns1.Name, GetStringFromChan(nsDeleteChan))
-	assert.Equal(t, "all", GetStringFromChan(rsDeleteChan))
```

Comment:
i was ok with this change since the code is now leveraging the shared deletion logic which itself has tests to verify the right set of operations were invoked.  duplicating that seemed unnecessary.  maybe a future test could just verify that the deleter itself was invoked?